### PR TITLE
License URI on exported Excel file

### LINKED ISSUE #400: License URI is missing on exported Excel file.
License URI is missing on exported Excel file.

### PR DESCRIPTION
**Pull request by lmsilvera**
License URI on exported Excel file

closes #400

---
This change is [Reviewable](https://reviewable.io/reviews/opensalt/opensalt/401)

### CODE REVIEW

**Comment by roverwolf** [comment]
If the export is changed then the import should be updated as well to match.

Note the pending PR #394 to update the code to use PHPSpreadsheet intead of PHPExcel (@JorgeLu1s can you review that soon as it also has a fix to import the prior version of the export).

**Comment by lmsilvera** [comment]
@roverwolf PR already updated and ready for review.

**Comment by roverwolf** [comment]
@lmsilvera It does not look like the Excel import matches what is now being exported.

**Comment by brandonopened** [comment]
@roverwolf yes I'm working on that right now. The export file looks different due to the more machine-readable changes on CF association but CfDoc and CfItem will be the same in a few minutes here including this change on the import template.

**Comment by brandonopened** [comment]
Or Luis for now just add that extra line to the current importer so this PR can merge.

**Comment by lmsilvera** [comment]
ok @brandonopened I'll do it.

File: src/App/Service/ExcelImport.php [hunk 1]
```diff
@@ -129,16 +130,45 @@ private function saveDoc(Worksheet $sheet): LsDoc
...
 
+    private function getLicence(Worksheet $sheet): LsDefLicence
+    {
+        $uri = $this->getCellValueOrNull($sheet, 14, 2);
+        $title = $this->getCellValueOrNull($sheet, 15, 2);
+        $licenceText = $this->getCellValueOrNull($sheet, 16, 2);
+
+        // Look for this licence locally
+        $licence = $this->getEntityManager()->getRepository(LsDefLicence::class)
+            ->findOneByUri($uri);
```
Comment (roverwolf) [inline src/App/Service/ExcelImport.php]: Should probably check if `$uri` is empty before trying to find a licence that matches (or trying to create one), as it is possible for there to be no licence.

Comment (lmsilvera) [reply src/App/Service/ExcelImport.php]: sure thing.

**Comment by lmsilvera** [comment]
@roverwolf PR ready

File: src/App/Service/ExcelImport.php (hunk 1 above)
Comment (roverwolf) [reply src/App/Service/ExcelImport.php]: It would be better to check if `$uri` is empty before using it to query the database, but it will work.

**Pull request merged**